ocaml/idl/ocaml_backend/taskHelper.ml

```diff
@@ -53,8 +53,8 @@ let make ~__context ~http_other_config ?(description="") ?session_id ?subtask_of
   ref, uuid
 
 let rbac_assert_permission_fn = ref None (* required to break dep-cycle with rbac.ml *)
-let assert_can_destroy ?(ok_if_no_session_in_context=false) ~__context task_id =
-  let assert_permission_task_destroy_any () =
+let assert_op_valid ?(ok_if_no_session_in_context=false) ~__context task_id =
+  let assert_permission_task_op_any () =
     (match !rbac_assert_permission_fn with
       | None -> failwith "no taskhelper.rbac_assert_permission_fn" (* shouldn't ever happen *) 
       | Some fn -> fn ~__context ~permission:Rbac_static.permission_task_destroy_any
```

Comment:
Even though the function was renamed to be more general, it still checks the specific permission for task.destroy (`Rbac_static.permission_task_destroy_any`).